project/AppDependencies.scala

```diff
@@ -3,13 +3,14 @@ import sbt._
 object AppDependencies {
 
   val compile: Seq[ModuleID] = Seq(
-    "uk.gov.hmrc"       %% "bootstrap-backend-play-28" % "7.12.0",
-    "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-28"        % "0.74.0"
+    "uk.gov.hmrc"       %% "bootstrap-backend-play-28" % "7.15.0",
```

Comment:
I know we don't do this at the moment in any of our services, but to aid future upgrades, thinking it's sensible that put the bootstrap version in a val like `private val hmrcBootstrapVersion = 7.15.0` that is used for both compile and test, likewise for the `hmrc-mongo` library, you can have a val like `hmrcMongoVersion = 1.1.0`